var/spack/repos/builtin/packages/elsi/package.py

```diff
@@ -64,6 +86,13 @@ class Elsi(CMakePackage, CudaPackage):
     with when("+use_external_superlu"):
         depends_on("superlu-dist+cuda", when="+cuda")
         depends_on("superlu-dist~cuda", when="~cuda")
+    with when("+enable_pexsi +use_external_pexsi"):
+        depends_on("pexsi+fortran")
```

Comment:
Just checking: when `superlu-dist` is a `pexsi` dependency? that's why if `pexsi` is built internally we don't need to specify anything for it?

Reply:
Yes, `superlu-dist` is a `pexi` dependency and bundled internally: https://gitlab.com/elsi_project/elsi_interface/-/blob/master/CMakeLists.txt?ref_type=heads#L128-137

I'm declaring it explicitly here to propagate the `cuda` variant.